test/test.ts

```diff
@@ -14,8 +14,16 @@ import * as diff from 'diff'
         console.log('loaded osm schema')
 
         let db = new Database(process.env.DATABASE_URL)
-        let formattedOutput = await typescriptOfSchema(db, 'osm', ['users'])
-        let outputFile = process.env.CIRCLE_ARTIFACTS + '/osm.ts'
+        let formattedOutput = await typescriptOfSchema(
+            db,
+            'osm',
+            ['users'],
+            extractCommand(
+                ['node', 'schemats', 'generate', '-c', 'postgres://secretUser:secretPassword@localhost/test', '-t', 'users', '-o', 'osm.ts'],
+                'postgres://secretUser:secretPassword@localhost/test'
+            )
+        )
+        let outputFile = (process.env.CIRCLE_ARTIFACTS || '.') + '/osm.ts'
```

Comment:
Please output the file `osm.ts` inside `test` directory. [Gitignore](https://github.com/SweetIQ/schemats/pull/11/files#diff-a084b794bc0759e7a6b77810e01874f2R342) can also be updated to reflect the change.